yarn-project/bb-prover/src/avm_proving.test.ts

```diff
@@ -137,11 +137,11 @@ describe('AVM WitGen, proof generation and verification', () => {
     TIMEOUT,
   );
 
-  // TODO: Investigate why does the prover throws an out-of-range exception
-  it.skip(
+  it(
     'Should prove that mutated timestamp does not match',
```

Comment:
Is this what we are trying to test here? It looks like using timestamp is just the "vehicle" to show that reverts are correctly proven? (if yes maybe modify the text)

Reply:
It is testing both properties:
- revert effectively occurs when passing the wrong timestamp
- AVM can prove this code path (namely with a revert)

Reply:
l adapted the test naming and mention the revert.